lib/network.js

```diff
@@ -32,6 +32,8 @@ function Network (options) {
     self.ignoreProcess  = (options.ignoreProcess ===  false) ? false : true;
     self.ignoreInstance = (options.ignoreInstance ===  false) ? false : true;
     self.redis          = options.redis;
+    self.hostName       = options.hostname  || hostname;
```

Comment:
The original module scope variable is called `hostName`, not `hostname`. And there's an additional space before `||`.

I think this line should read `self.hostName       = options.hostname || hostName;`